### PR TITLE
Responsive Error Page

### PR DESCRIPTION
**Pull request by thomasHolman**

### CODE REVIEW

File: resources/error.html [hunk 1]
```diff
@@ -8,29 +8,34 @@
...
-        .row-fluid .span1{width:40.372340425532%;}
-        .pull-right{float:right;}
-        .ellipsis{overflow: hidden;white-space: nowrap;text-overflow: ellipsis;-o-text-overflow: ellipsis;}
+        .wrapper{padding: 3% 6%; margin:0 auto 5em;}
+        @media (min-width: 768px) {
+            html{font-size:100%;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;font-family:sans-serif;color:#222;}
+            body{font-size:10pt;}
+            a{color:#00e;}
+            .wrapper{max-width: 768px}
+            .error-spacer{height:4em;}
```
Comment (jbrooksuk) [inline resources/error.html]: @GrahamCampbell if you merge this, you'll want to remove this line.

**Comment by GrahamCampbell** [comment]
Tests are failing.

**Comment by jbrooksuk** [comment]
I'll fix the tests up now.

File: resources/error.html (hunk 1 above)
Comment (GrahamCampbell) [inline resources/error.html]: @jbrooksuk Could you sort that too please? :)

**Comment by jbrooksuk** [comment]
- Tests fixed.
- Added to changelog
- Removed the `.error-spacer`

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request merged**